### PR TITLE
FSx Filesystem changes

### PR DESCRIPTION
**Pull request by shyamkumarlchauhan**
https://dimagi-dev.atlassian.net/browse/SAAS-14074

##### ENVIRONMENTS AFFECTED
staging

### CODE REVIEW

**Review by dannyroberts — APPROVED** [review]

File: src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml [hunk 1]
```diff
@@ -101,14 +101,40 @@
...
+    state: directory
+    owner: "{{ cchq_user }}"
+    group: "{{ cchq_user }}"
+    mode: "0755"
+  tags:
+    - localsettings
+    - formplayer_deploy
+  with_items:
     - "{{ formplayer_data_dir }}"
+  when: not check_data_dir
```
Comment (snopoke) [inline src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml:132]: @shyamkumarlchauhan why is this necessary? Ansible should be handling this automatically.

Comment (shyamkumarlchauhan) [reply src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml:132]: as this will be a NFS mount so ownership/mode change of a NFS mount may generate error if cchq_uid/cchq_gid is not defined in public.yml

**Comment by dannyroberts** [comment]
I just noticed
> [cchq uid/gid update](https://github.com/dimagi/commcare-cloud/pull/5642/commits/124b354a7f025be48edc855d3f9106a417260bb9)
wasn't reviewed before merging. @shyamkumarlchauhan reminder to make sure the last commit(s) are also reviewed before merge if there are commits added after an approval.

And to close the loop, no issues with that commit, approved ✅

**Comment by shyamkumarlchauhan** [comment]
> I just noticed
> 
> > [cchq uid/gid update](https://github.com/dimagi/commcare-cloud/pull/5642/commits/124b354a7f025be48edc855d3f9106a417260bb9)
> 
> wasn't reviewed before merging. @shyamkumarlchauhan reminder to make sure the last commit(s) are also reviewed before merge if there are commits added after an approval.
> 
> And to close the loop, no issues with that commit, approved ✅

Noted down, will ensure to get it reviewed before next merge.
